### PR TITLE
feat(pie-lottie-player): DSW-2365 add pie-lottie-player

### PR DESCRIPTION
**Pull request by fernandofranca**
- Adds pages for the new component for the vanilla, next 13, next 14 and nuxt apps
- Adds the `PERCY=true` as URL parameter to visual tests, so we can switch components behaviour when performing VRT
- Creates the possibility of adding a pause before taking screenshots

### CODE REVIEW

**Review by kevinrodrigues — DISMISSED** [review]

File: nextjs-app-v13/src/pages/components/lottie-player.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import NavigationLayout from "@/layout/navigation";
+import { PieLottiePlayer } from '@justeattakeaway/pie-lottie-player/dist/react.js';
+import { PieButton } from '@justeattakeaway/pie-button/dist/react.js';
+import { useState } from "react";
+import { isServer } from 'lit';
+
+const animations = [
+    "/animations/preparing.json",
+    "/animations/courier.json",
```
Comment (xander-marjoram) [inline nextjs-app-v13/src/pages/components/lottie-player.tsx:9]: On the Next 13 app when I press the "load another animation" button, it loads the courier animation but it doesn't play, same again for the order confirmed animation.

Comment (xander-marjoram) [reply nextjs-app-v13/src/pages/components/lottie-player.tsx:9]: Doing this in the console manages to get it playing
`document.querySelector('pie-lottie-player').play()`
so it is loading correctly, just not autoplaying for some reason

File: nuxt-app/package.json [hunk 2]
```diff
@@ -22,6 +22,7 @@
     "@justeattakeaway/pie-cookie-banner": "0.26.3",
     "@justeattakeaway/pie-css": "0.12.1",
     "@justeattakeaway/pie-icons-webc": "0.25.1",
+    "@justeattakeaway/pie-lottie-player": "0.0.1",
```
Comment (xander-marjoram) [inline nuxt-app/package.json:25]: Does it still work when imported from `pie-webc`?

File: nextjs-app-v13/src/pages/components/lottie-player.tsx (hunk 1 above)
Comment (fernandofranca) [reply nextjs-app-v13/src/pages/components/lottie-player.tsx:9]: Good catch!

Comment (fernandofranca) [reply nextjs-app-v13/src/pages/components/lottie-player.tsx:9]: Fixed it here: https://github.com/justeattakeaway/pie/pull/1845

File: nuxt-app/package.json (hunk 2 above)
Comment (fernandofranca) [reply nuxt-app/package.json]: Good point!

File: nuxt-app/nuxt.config.ts [hunk 3]
```diff
@@ -6,6 +6,7 @@ export default defineNuxtConfig({
       moduleSideEffects: [
           '@justeattakeaway/pie-icons-webc',
           '@justeattakeaway/pie-webc',
+          '@justeattakeaway/pie-lottie-player',
```
Comment (xander-marjoram) [inline nuxt-app/nuxt.config.ts:9]: This might no longer be needed if we're pulling it in from pie-webc

**Review by xander-marjoram — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Pull request merged**